devvit.json

```diff
@@ -5,6 +5,10 @@
     "dir": "dist/client",
     "entrypoints": {
       "default": {
+        "inline": true,
+        "entry": "splash/splash.html"
+      },
+      "game": {
         "entry": "index.html"
```

Comment:
:no_bell: since this is no longer the default entry, suggest making this game.html and matching splash organization pattern. I'm thinking either:

1. entries/splash.html and entries/game.html.
2. splash/splash.html and game/game.html.